.travis.yml

```diff
@@ -19,6 +19,8 @@ addons:
 
 # We use matrix include so that xspec builds (which are more complicated and long) are started first
 matrix:
+  allow_failures:
+      - python: "3.5"
```

Comment:
Thanks @bsipocz. Not sure why I kept the `allow_failures` line, since I am happy with the build to be red until all the tests pass. I'll remove that line altogether, rather than fixing it.